infra/discovery/devops/es-cli.js

```diff
@@ -164,7 +166,8 @@ function printInteractiveUsage() {
 3 create alias
 4 delete alias
 5 delete index
-6 reindex
+6 reindex (copy to new index)
+7 rebuild index (copy to temp index, and copy back)
```

Comment:
Do we actually need this ?

We use alias in prod to point the search queries to a particular index. 
Assuming we have
 - index "listing1"
 - alias "listing" pointing to listing1

We can with the existing es-cli functionality do:
 - re-index from "listing1" into new index "listing2"
 - update alias "listing" to point to "listing2"

Reply:
We don't need it in production, but it's handy for me when working locally with docker.

It is a near duplication, it just speeds up my particular use.